INSTALL

```diff
@@ -0,0 +1,368 @@
+Installation Instructions
```

Comment:
We should probably edit this file sometime. The default Autoconf one is pretty bland and generic.

I checked it in, in preparation for that, and also to try working around an annoying issue in automake (see other comments on "foreign" flag).